addon-api/content-script/Auth.js

```diff
@@ -1,11 +1,10 @@
-import AuthCommon from "../common/Auth.js";
-
-export default class Auth extends AuthCommon {
-  constructor(...args) {
-    super(...args);
-    this._refresh();
-  }
+import FetchableAuth from "../common/FetchableAuth.js";
 
+/**
+ * Authentication related utilities.
+ * @extends FetchableAuth
+ */
+export default class Auth extends FetchableAuth {
```

Comment:
It looks like this is empty except what's inherited from FetchableAuth now
..why?

Reply:
Oh I see, _getCookie is different in each subclass. Please add _getCookie to FetchableAuth and have it throw a "Subclasses must implement this" error.

Reply:
Oh I see, _getCookie is different in each subclass. Please add _getCookie to FetchableAuth and have it throw a "Subclasses must implement this" error.